projects/igniteui-angular/src/lib/buttonGroup/buttonGroup.component.ts

```diff
@@ -349,6 +356,17 @@ export class IgxButtonGroupComponent implements AfterViewInit, OnDestroy {
         }
     }
 
+    /**
+     * @hidden
+     */
+    public ngAfterContentInit() {
+        this.templateButtons.forEach( (button) => {
```

Comment:
I think we should update the display density of the `viewButtons` not the `templateButtons`.